src/components/actions/coreActions.ts

```diff
@@ -336,6 +336,10 @@ const coreActions: CoreActions = {
           operator: 'objectProperties',
           children: ['outputCumulative.updatedResponses'],
         },
+        reviewId: {
+          operator: 'objectProperties',
+          children: ['applicationData.reviewData.reviewId'],
+        },
```

Comment:
You shouldn't need this -- it gets if from `applicationData` if present.

Reply:
Every action checks applicationData for applicationId and reviewId. You only need to provide it is a parameter if you're wanting to override the current value (which we would rarely want to do).

Reply:
It's either this or we need to set `triggerBy` REVIEW, I choose to pass the `reviewId`

Reply:
That's what I'm saying-- you don't need to pass it, it's already there (look at the reviewId=... at the top).

Reply:
I'm confused -- this is for review submissions right?

I really don't see how you passing "applicationData.reviewData.reviewId" would be any different to it getting it automatically from the same applicationData. If it _is_ different, then there's something more fundamentally wrong and we should be fixing that.